src/test/java/com/daml/extensions/testing/junit5/SandboxIT.java

```diff
@@ -40,9 +39,4 @@ public void specifiedPortIsAssignedWhenSandboxIsStarted() {
   public void ledgerIdSpecified() {
     assertThat(sandbox.getLedgerId(), is("sample-ledger"));
   }
-
-  private void assertsIsBetween(int x, int low, int high) {
```

Comment:
@vshneer why is this deleted?

Reply:
there was a test that checked if auto assigned port is between a range boundaries. 
I put a new test instead of it - check if custom port assigned. 
Because I cannot test these two in one test class - sandbox port either custom or auto assigned...
Maybe I should return this boundary test back, and for custom test create another class. I'll do it

Reply:
I would advise having two separate tests (classes) to test them, then.